### DISCUSSION TITLE
Aeotec ZGA001 Zigbee range extender compatibility

### DISCUSSION THREAD

**Discussion by expresscisco** [Koenkk/zigbee2mqtt#14267] [Devices]
Hello from Versailles, France

I am completely new on all of this. I hope I am posting in the right place. I did a prior search but it seems the device I purchased is quite new on the market. 

I am using z2m and need to get a better link quality for my leak sensors all over my house. I bought a Aeotec ZGA001 which is recognised by zha but not z2m and would like to know if there is a chance it gets supported. 

I would be ok to help but I have no clue how. 

When I start pairing with z2m it sees the device but it’s not supported. 

Merci !

**Comment by FROCHE38** [comment]
Hello from Grenoble, France

Same problem Aeotec ZGA001 recognised by zha but not by z2m. Please add support
![image](https://github.com/Koenkk/zigbee2mqtt/assets/37023141/738a9a88-9a7d-412f-b3c6-365c0eea3990)

**Comment by FROCHE38** [comment]
Hello,
This is always not fixed with version zigbee2mqtt 1.40.0. Can someone help us?

**Reply by chris-1243** [reply]
Have a look there

https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

**Comment by 1mfaasj** [comment]
Ive got the same issue
But this should be reported as an issue, not a discussion topic?

**Comment by FROCHE38** [comment]
Hello Chris.
We cannot go to the dév console. We have an error message. Then we are
unable to follow thé procedure

Le mar. 10 sept. 2024 à 23:11, chris-1243 ***@***.***> a
écrit :

**Reply by chris-1243** [reply]
"In case your device does not report anything out of the blue, the Clusters page at Zigbee2MQTT's frontend (found under the device in the dashboard) also exposes the clusters."

What is the output of this ?

**Reply by 1mfaasj** [reply]
The output here is 0x8c65a3fffe5216b0
Do we need this for something?

I also created an issue: https://github.com/Koenkk/zigbee2mqtt/issues/23916

**Reply by chris-1243** [reply]
I would have expected something like this 

![Untitled](https://github.com/user-attachments/assets/173016d1-4521-4aa3-9104-bf38e7957da9)
